client/src/views/Settings/Tabs/EditProfile/index.vue

```diff
@@ -1,140 +1,251 @@
 <template>
-  <q-page class="q-pa-lg">
+  <q-page>
     <q-form
       class="q-gutter-md"
       @submit="onSubmit"
       @reset="onReset"
     >
-      <div>
-        <avatar :img="photoUrl" />
-        <q-input
-          @input="uploadAvatar"
-          type="file"
-          class="inputFile"
-        />
-      </div>
-      <div>
-        <img
-          :src="backgroundUrl"
-          class="backgroundUrl"
-        />
-        <q-input
-          @input="uploadBackgroundUrl"
-          type="file"
-          class="inputFile"
-        />
-      </div>
-      <q-input
-        label="Username"
-        placeholder="Add your username"
-        v-model="profile.nickname"
-        :rules="[requiredField, checkNicknameField]"
-      />
-      <q-input
-        label="Bio"
-        type="textarea"
-        placeholder="Add a bio to your profile"
-        v-model="profile.bio"
-        autogrow
-        counter
-        :rules="[checkUserDescriptionField]"
-      />
-      <q-input
-        label="Status"
-        type="textarea"
-        placeholder="Add a status to your profile"
-        v-model="profile.status"
-        autogrow
-        counter
-        :rules="[checkUserDescriptionField]"
-      />
-      <q-select
-        label="Interests"
-        v-model="profile.interests"
-        use-input
-        use-chips
-        multiple
-        input-debounce="0"
-        @new-value="createValue"
-        :options="filterOptions"
-        @filter="filterFn"
-      />
-      <q-input
-        label="YouTube"
-        placeholder="Add YouTube link to your profile"
-        v-model="profile.socialAccounts.youtube"
-        type="url"
-      />
-      <q-input
-        label="Instagram"
-        placeholder="Add Instagram link to your profile"
-        v-model="profile.socialAccounts.instagram"
-        type="url"
-      />
-      <q-input
-        label="Facebook"
-        placeholder="Add Facebook link to your profile"
-        v-model="profile.socialAccounts.facebook"
-        type="url"
-      />
+
+      <q-card class="my-card">
+        <q-card-section>
+          <div class="text-h6">Avatar</div>
+        </q-card-section>
+        <q-card-section>
+          <template v-if="photoUrl===''">
```

Comment:
I'm a bit confused why you check if photoUrl equals to empty string and inside that block you pass to avatar component :img  which is empty string.